app/code/Magento/SalesRule/Model/ResourceModel/Rule/DeleteCustomerUsage.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\SalesRule\Model\ResourceModel\Rule;
+
+class DeleteCustomerUsage
+{
+    /** @var \Magento\SalesRule\Model\ResourceModel\Rule\Customer */
+    protected $_customerRuleDetails;
```

Comment:
Please, make this var private and try to avoid underscores in method and variable names

Reply:
@sidolov , i have made changes. kindly check and let me know.